qiskit/pulse/schedule.py

```diff
@@ -16,16 +16,17 @@
 """
 
 import abc
+
 import copy
 import itertools
 import multiprocessing as mp
 import sys
 from typing import List, Tuple, Iterable, Union, Dict, Callable, Set, Optional
 
 from qiskit.circuit.parameterexpression import ParameterExpression, ParameterValueType
+from qiskit.pulse import instructions
```

Comment:
This import is unnecessary and is causing circular import/linting issues.

Reply:
Thank you for that. Hope it passes this time around

Reply:
```
Warning, treated as error:
Cannot resolve forward reference in type annotations of "qiskit.pulse.Schedule": name 'Instruction' is not defined
___________________________________ summary ____________________________________
ERROR:   docs: commands failed
```

This is the error Azure build gives me. The problem is with the forward References in Schedule and Instruction. According to [PEP484](https://www.python.org/dev/peps/pep-0484/#forward-references) forward references section, this would cause a circular dependency either way. Can you help resolve this problem?